redis/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Is `log4j.properties` file necessary in calcite distribution ?

Reply:
I agree it does not sound right to distribute log4j.properties

Reply:
> I agree it does not sound right to distribute log4j.properties

well, I will delete this log configuration file.